src/service/db.ts

```diff
@@ -7,6 +7,10 @@ export async function getTable(tableName: string) {
   })
 }
 
-export async function getItem(id: number) {
-  return invoke('get_item', { id }) as Promise<Item>
+export async function getItem(id: number):Promise<Item> {
+  return await invoke('get_item', { id }).catch(() => null) as Promise<Item>
+}
+
+export async function getItems():Promise<Item[]>{
+  return await invoke('get_items')
```

Comment:
should we have a fallback here? maybe we should have catch's on all db functions just to make sure that an empty items table in the db doesn't break things - we haven't done any testing to see what happens in that case.